alpha/apps/kaltura/modules/extwidget/actions/embedPlaykitJsAction.class.php

```diff
@@ -339,29 +339,27 @@ private function mergeVersionsParamIntoConfig()
 	
 	private function setLatestOrBetaVersionNumber()
 	{
-		if ($this->bundleConfig) {
-			//if latest/beta version required set version number in config obj
-			$isLatestVersionRequired = array_search("{latest}", $this->bundleConfig) !== false;
-			$isBetaVersionRequired = array_search("{beta}", $this->bundleConfig) !== false;
+		//if latest/beta version required set version number in config obj
+		$isLatestVersionRequired = array_search("{latest}", $this->bundleConfig) !== false;
```

Comment:
Move the strings "{latest}" and "{beta}" to be class constants since you use them more than once.